dashboard/modules/logical_view/logical_view_head.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import aiohttp.web
+import ray.utils
+import ray.new_dashboard.utils as dashboard_utils
+import ray.new_dashboard.actor_utils as actor_utils
+from ray.new_dashboard.utils import rest_response
+from ray.new_dashboard.datacenter import DataOrganizer
+from ray.core.generated import core_worker_pb2
+from ray.core.generated import core_worker_pb2_grpc
+
+from grpc.experimental import aio as aiogrpc
+
+logger = logging.getLogger(__name__)
+routes = dashboard_utils.ClassMethodRouteTable
+
+
+class LogicalViewHead(dashboard_utils.DashboardHeadModule):
+    @routes.get("/logical/actor_groups")
+    async def get_actor_groups(self, req) -> aiohttp.web.Response:
+        actors = await DataOrganizer.get_all_actors()
+        actor_creation_tasks = await DataOrganizer.get_actor_creation_tasks()
+        # actor_creation_tasks have some common interface with actors,
+        # and they get processed and shown in tandem in the logical view
+        # hence we merge them together before constructing actor groups.
+        actors.update(actor_creation_tasks)
+        actor_groups = actor_utils.construct_actor_groups(actors)
+        return await rest_response(
+            success=True,
+            message="Fetched actor groups.",
+            actor_groups=actor_groups)
+
+    @routes.get("/logical/kill_actor")
+    async def kill_actor(self, req) -> aiohttp.web.Response:
+        try:
+            actor_id = req.query["actorId"]
+            ip_address = req.query["ipAddress"]
+            port = req.query["port"]
+        except KeyError:
+            return await rest_response(success=False, message=f"Bad Request")
+        try:
+            channel = aiogrpc.insecure_channel(f"{ip_address}:{port}")
+            stub = core_worker_pb2_grpc.CoreWorkerServiceStub(channel)
+
+            await stub.KillActor(
+                core_worker_pb2.KillActorRequest(
+                    intended_actor_id=ray.utils.hex_to_binary(actor_id)))
+
+        except aiogrpc.AioRpcError:
+            # This always throws an exception because the worker
+            # is killed and the channel is closed on the worker side
+            # before this handler, however it deletes the actor correctly.
```

Comment:
Thanks for the comments!

Reply:
Can we actually check `ray.kill` API and see how it handles the issue?